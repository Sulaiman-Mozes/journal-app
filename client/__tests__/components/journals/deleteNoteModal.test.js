import React from 'react';
import expect from 'expect';
import { shallow } from 'enzyme';
import DeleteNoteModal from '../../../components/journals/deleteModal';

describe('Provider and DeleteNoteModal', () => {
  const props = {
    handleDelete: jest.fn(),
  };

  const DeleteNote = shallow(<DeleteNoteModal {...props} />);

  it('renders <Provider/> correctly', () => {
    expect(DeleteNote).toMatchSnapshot();
  });
});
