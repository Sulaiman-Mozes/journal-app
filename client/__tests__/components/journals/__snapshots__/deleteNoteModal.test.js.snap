// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Provider and DeleteNoteModal renders <Provider/> correctly 1`] = `
<div
  aria-hidden="true"
  aria-labelledby="exampleModalLabel"
  className="modal fade"
  id="modalConfirmDelete"
  role="dialog"
  tabIndex="-1"
>
  <div
    className="modal-dialog modal-sm modal-notify modal-danger"
    role="document"
  >
    <div
      className="modal-content text-center"
    >
      <div
        className="modal-header d-flex justify-content-center"
      >
        <p
          className="heading"
        >
          Are you sure?
        </p>
      </div>
      <div
        className="modal-body"
      >
        <i
          className="fas fa-times fa-4x animated rotateIn"
        />
      </div>
      <div
        className="modal-footer flex-center"
      >
        <button
          className="btn  btn-outline-danger"
          data-dismiss="modal"
          onClick={[MockFunction]}
          type="button"
        >
          Yes
        </button>
        <button
          className="btn  btn-danger waves-effect"
          data-dismiss="modal"
          type="button"
        >
          No
        </button>
      </div>
    </div>
  </div>
</div>
`;
